src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -177,14 +177,14 @@ def __init__(self, username, password, name=None, email=None, roles=None,
             self.roles = set()
         else:
             self.roles = roles
-        if lastUpdate is None:
-            self.refreshLastUpdate()
+        if last_update is None:
+            self.refresh_last_update()
         else:
-            self.lastUpdate = lastUpdate
+            self.last_update = last_update
```

Comment:
```
2019-08-25T05:50:47.201 INFO:tasks.ceph.mgr.x.smithi060.stderr:[25/Aug/2019:05:50:47] HTTP Traceback (most recent call last):
2019-08-25T05:50:47.202 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/lib/python2.7/site-packages/cherrypy/_cprequest.py", line 656, in respond
2019-08-25T05:50:47.202 INFO:tasks.ceph.mgr.x.smithi060.stderr:    response.body = self.handler()
2019-08-25T05:50:47.202 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/lib/python2.7/site-packages/cherrypy/lib/encoding.py", line 188, in __call__
2019-08-25T05:50:47.202 INFO:tasks.ceph.mgr.x.smithi060.stderr:    self.body = self.oldhandler(*args, **kwargs)
2019-08-25T05:50:47.202 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/lib/python2.7/site-packages/cherrypy/_cptools.py", line 221, in wrap
2019-08-25T05:50:47.202 INFO:tasks.ceph.mgr.x.smithi060.stderr:    return self.newhandler(innerfunc, *args, **kwargs)
2019-08-25T05:50:47.202 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/share/ceph/mgr/dashboard/services/exception.py", line 88, in dashboard_exception_handler
2019-08-25T05:50:47.203 INFO:tasks.ceph.mgr.x.smithi060.stderr:    return handler(*args, **kwargs)
2019-08-25T05:50:47.203 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/lib/python2.7/site-packages/cherrypy/_cpdispatch.py", line 34, in __call__
2019-08-25T05:50:47.203 INFO:tasks.ceph.mgr.x.smithi060.stderr:    return self.callable(*self.args, **self.kwargs)
2019-08-25T05:50:47.203 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/share/ceph/mgr/dashboard/controllers/__init__.py", line 645, in inner
2019-08-25T05:50:47.203 INFO:tasks.ceph.mgr.x.smithi060.stderr:    ret = func(*args, **kwargs)
2019-08-25T05:50:47.203 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/share/ceph/mgr/dashboard/controllers/__init__.py", line 838, in wrapper
2019-08-25T05:50:47.203 INFO:tasks.ceph.mgr.x.smithi060.stderr:    return func(*vpath, **params)
2019-08-25T05:50:47.204 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/share/ceph/mgr/dashboard/controllers/auth.py", line 58, in check
2019-08-25T05:50:47.204 INFO:tasks.ceph.mgr.x.smithi060.stderr:    user = JwtManager.get_user(token)
2019-08-25T05:50:47.204 INFO:tasks.ceph.mgr.x.smithi060.stderr:  File "/usr/share/ceph/mgr/dashboard/services/auth.py", line 89, in get_user
2019-08-25T05:50:47.204 INFO:tasks.ceph.mgr.x.smithi060.stderr:    if user.lastUpdate <= dtoken['iat']:
2019-08-25T05:50:47.205 INFO:tasks.ceph.mgr.x.smithi060.stderr:AttributeError: 'User' object has no attribute 'lastUpdate'
```

might want to update `src/pybind/mgr/dashboard/services/auth.py` accordingly.

see http://pulpito.ceph.com/kchai-2019-08-25_05:28:04-rados-wip-kefu-testing-2019-08-23-1454-distro-basic-smithi/4251285/

Reply:
Thx, good spotting. This code has been changed by a previous PR.